config/application.rb

```diff
@@ -18,5 +18,6 @@ class Application < Rails::Application
     require Rails.root.join('lib/custom_public_exceptions')
     config.exceptions_app = CustomPublicExceptions.new(Rails.public_path)
     config.middleware.insert_after ActionDispatch::Static, Rack::Deflater
+    config.view_component.preview_paths << Rails.root.join('spec/components/previews')
```

Comment:
If the route is only going to be in dev, can move this to the dev env config file